plejd/ble.bluez.js

```diff
@@ -683,4 +683,4 @@ class PlejdService extends EventEmitter {
   }
 }
 
-module.exports = PlejdService;
\ No newline at end of file
+module.exports = PlejdService;
```

Comment:
This was un-intentional but I did the edit in Github browser edit so I blame Github